kolibri/core/auth/models.py

```diff
@@ -742,7 +742,7 @@ def has_morango_certificate_scope_permission(
             target_user = FacilityUser.objects.get(id=scope_params.get("user_id"))
             if self == target_user:
                 return True
-            if self.has_role_for_user(target_user, role_kinds.ADMIN):
+            if self.has_role_for_user(role_kinds.ADMIN, target_user):
```

Comment:
I feel like this PR could be limited to just this change, and then the need to make the above changes removed by changing in this line: https://github.com/learningequality/kolibri/pull/8185/files#diff-c968516236d8c64ebcae34e7fb63ce29d0d217e6642e416de5386a14a362ceddR54 to `facility` instead of `facility_id`.

Reply:
Happy with it if we can change https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/auth/management/utils.py#L238-L249 to work with `facility`